src/olympia/files/migrations/0029_auto_20230301_1843.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.18 on 2023-03-01 18:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('files', '0028_set_approval_date'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='webextpermission',
+            name='host_permissions',
+            field=models.JSONField(default=dict),
```

Comment:
We usually add new columns with `null=True, default=None` for backwards-compatibility with the previous tag, because Django doesn't add database defaults in non-null cases, but it looks like it does for `JSONField`, TIL. That generates:
```
ALTER TABLE `webext_permissions` ADD COLUMN `host_permissions` json DEFAULT ('{}') NOT NULL;
```

Which _is_ backwards-compatible, so that's fine. Carry on.